src/xdocs/config_javadoc.xml

```diff
@@ -3311,14 +3311,20 @@ public boolean testMethod(int firstParam) {
               <td>8.3</td>
             </tr>
             <tr>
-              <td>ignoredTags</td>
+              <td>
+                <a href = "http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDBEFIF">
+                ignoredTags</a>
+              </td>
               <td>Specify block tags which are ignored by the check.</td>
```

Comment:
please make a link over `block tags` text in description. The same for other property.

Reply:
Done.

I tried doing this before. While, XdocsPagesTest is not failing anymore. I still get an error with XdocsJavaDocsTest.

https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=1249&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=296

<img width="1421" alt="Screen Shot 2021-02-19 at 9 34 31 AM" src="https://user-images.githubusercontent.com/31570659/108525524-c8d49280-7295-11eb-9983-b61559e0b23f.png">

I don't understand why `Actual   :Specify block tags which are ignored by the check.` is shown in the test. Even though I have included the link in the source code for SingleLineJavadocCheck.java file.

Reply:
Please try to update text over setter.

Reply:
Sorry, I did not understand what you meant by "update text over setter".

Reply:
Javadoc over setter method for this field.
`setIgnoredTags"

Reply:
Thank you for the clarification!
I also had to update the text over variable declaration along with the setter to get the test passing.